app/Schema/bank_account_schema.py

```diff
@@ -1,25 +1,66 @@
 from app import ma
+from marshmallow.validate import Length
 from marshmallow import fields, INCLUDE
+from random import randint
 
 
 class BankAccountSchema(ma.Schema):
     """Adding the Schema validations to the BankAccount """
-
-    account_number = fields.Int(required=True)
-    is_active = fields.Boolean(required=True)
-    deleted = fields.Boolean(required=True)
-    user_id = fields.Boolean(required=True)
+    user_id = fields.Int(required=True)
     account_type_id = fields.Int(required=True)
     branch_id = fields.Int(required=True)
 
     class Meta:
 
         """ Exposed fields """
 
-        fields = ('account_number', 'is_active', 'deleted', 'user_id', 'account_type_id', 'branch_id')
+        fields = ('account_number', 'is_active', 'deleted', 'user_id', 'account_type_id', 'branch_id', 'created_on')
         unknown = INCLUDE
         load_instance = True
 
 
 bank_account_schema = BankAccountSchema()
 bank_accounts_schema = BankAccountSchema(many=True)
+
+
+def random_account_number(n):
```

Comment:
function name should be `generate_random_number`. You are appending this random number with branch id and then account number is formed.